### ISSUE TITLE
Change behaviour of tooltip

### ISSUE THREAD

**Issue by efb4f5ff-1298-471a-8973-3d47447115dc** [FreeTubeApp/FreeTube#1805]
**Is your feature request related to a usage problem (not a bug)? Please describe.**
In my opinion the behavior of the tooltip is very user unfriendly. When hover over the tooltip the messages appears but when u try to hover to the message it disappears. When u hover on the tooltip and click on it u can hover the message but when u want to copy the message u cant and the message disappear.

**Describe the solution you'd like to see implemented**
So i would opt for removing hovering over the Toolip ? to reveal the message. Instead i would just like to click on it and the message will appear and click on it again to let the message disappear. When message X is opened and u click on Tooltip Y to see message Y. Message Y opens and Message X disappears.

When u hover over the message u must be able to copy the text.

**Describe alternatives you've considered**
None.

**Video**

https://user-images.githubusercontent.com/73130443/136675823-99a052d7-c911-4bcc-bdb0-278719440705.mp4

**Additional context**
None.

**Comment by jasonhenriquez** [comment]
I agree that we need a usability improvement here, but I don't agree with the suggestion. Tooltips should open on hover, but they also should stay open when you hover onto the text & allow you to copy, which we're not doing.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
That's a way better solution! In retrospect looking back at this old issue of mine my suggested solution isn't that good😬

**Comment by jasonhenriquez** [comment]
I get why you phrased it like that, because we have our tooltips spatially disconnected from the the actual icon, which is non-standard. To have my suggested behavior, you would either have to make the tooltip popup be touching the icon at the edge, or have a larger non-visible `div` appear on `:hover` that stays open on `:hover` & allows the cursor to reach the popup easily.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Can't wait to see this changed! There has been multiple occasions i wanted to copy the tooltip message and send it to people
